### PR TITLE
add whatsapp, update pinterest

### PR DESCRIPTION
**Pull request by xiwcx**
https://jira.nationalgeographic.com/browse/SYTO-450

- pull the branch
- run `grunt serve` (make sure to run `bower install && npm install && bundle install` *first* if you've never run this branch before)
- if everything looks good merge and run `grunt deploy`

### CODE REVIEW

File: app/vectors/pinterest.svg [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+<!-- Generator: Adobe Illustrator 18.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 width="32px" height="32px" viewBox="0 0 32 32" enable-background="new 0 0 32 32" xml:space="preserve">
-<path d="M13.787,21.165C12.946,25.571,11.92,29.794,8.881,32c-0.94-6.658,1.376-11.658,2.453-16.968
-	c-1.836-3.086,0.22-9.302,4.089-7.769c4.759,1.882-4.123,11.477,1.84,12.675c6.225,1.251,8.766-10.8,4.905-14.719
-	C16.591-0.439,5.935,5.089,7.245,13.191c0.318,1.981,2.364,2.581,0.817,5.314c-3.569-0.789-4.635-3.604-4.498-7.358
-	C3.784,5.004,9.085,0.703,14.4,0.107c6.722-0.752,13.029,2.469,13.901,8.792c0.98,7.136-3.035,14.866-10.223,14.31
-	C16.131,23.059,15.313,22.094,13.787,21.165z"/>
+	 viewBox="0 0 32 32" enable-background="new 0 0 32 32" xml:space="preserve">
```
Comment (alexrqs) [inline app/vectors/pinterest.svg]: I notice this new files are missing width and height and all the other icons on the folder have those, any reason?

File: app/vectors/whatsapp.svg [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 32 32" enable-background="new 0 0 32 32" xml:space="preserve">
+<g>
+	<g>
+		<g>
+			<path fill-rule="evenodd" clip-rule="evenodd" d="M27.448,4.522c-2.984-2.986-6.952-4.632-11.18-4.633
```
Comment (alexrqs) [inline app/vectors/whatsapp.svg]: ``fill-rule="evenodd"`` & ``clip-rule="evenodd"`` aren't necessary.

**Comment by alexrqs** [comment]
Any aditional steps to run this branch?

I'm having this output:

Running "sass:fontcustom" (sass) task
> Source file ".tmp/icongs.scss" not found.

- Running ``"browser_sync:serve"`` (browser_sync) task
- Warning: Cannot read property 'prototype' of undefined Use --force to continue.

Aborted due to warnings.

**Comment by pmourelle** [comment]
@alexrqs before that you probably have:
`Running "shell:fontcustom" (shell) task
       error  Please install fontforge first. Visit <http://fontcustom.com> for instructions.`
That's the real issue.

**Comment by pmourelle** [comment]
@alexrqs @xiwcx the issue with this is that that URL points to an abandoned (and for sale) website.
This would be the URL: http://fontcustom.github.io/fontcustom/#installation

**Comment by pmourelle** [comment]
@xiwcx after installing everything, `grunt serve` ends with error:
```
Running "shell:fontcustom" (shell) task
      status  Forcing compile.
      create  .tmp/icongs.ttf
              .tmp/icongs.svg
              .tmp/icongs.woff
              .tmp/icongs.eot
      create  .tmp/_icongs.scss
              .tmp/index.html

Running "copy:fontcustom" (copy) task
Copied 1 files

Running "sass:fontcustom" (sass) task

Running "svgstore:icongs" (svgstore) task
File .tmp/icongs.svg created.

Running "browser_sync:serve" (browser_sync) task
Warning: Cannot read property 'prototype' of undefined Use --force to continue.

Aborted due to warnings.
```

**Comment by chuckcarpenter** [comment]
```
Running "browser_sync:serve" (browser_sync) task
Warning: Cannot read property 'prototype' of undefined Use --force to continue.
```

same issue for me, I updated the browser-sync, it's fixed but then I have other errors where all the files aren't copying over to .tmp

```

**Pull request merged**